website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -7,36 +7,34 @@ description: >
 ---
 
 Karpenter sets a Kubernetes [finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/) on each node it provisions.
-The finalizer specifies additional actions the Karpenter controller will take in response to a node deletion request.
-These include:
-
-* Marking the node as unschedulable, so no further pods can be scheduled there.
-* Evicting all pods other than daemonsets from the node.
-* Terminating the instance from the cloud provider.
-* Deleting the node from the Kubernetes cluster.
+The finalizer signals to the Karpenter Termination controller to gracefully shutdown a node that receives a deletion request. Karpenter deprovisioning can be executed by Karpenter through the Deprovisioning Controler or by the user through manual deprovisioning.
+
+### Deprovisioning Controller
+Karpenter automatically discovers deprovisionable nodes and spins up replacements when needed. Karpenter deprovisions nodes by executing one [automatic method](#methods) at a time, in order of Expiration, Drift, Emptiness, then Consolidation. Each method varies slightly but they all follow the standard deprovisioning process:
+1. Generate a list of deprovisionable nodes and sort them as needed.
+   1. If there are no deprovisionable nodes, continue to the next deprovisioning method.
+2. For each deprovisionable node, execute a scheduling simulation with the pods on the node to find if any replacement nodes are needed.
+3. Cordon to-be-deprovisioned node(s) so no further pods can be scheduled on the node.
```

Comment:
You're using `to-be-deprovisioned` a lot and, to me, it feels hard to read. Can we use something shorter or change this to be something like "Initiate deprovisioning by cordoning the node(s) so no further pods can be scheduled on the node."

Reply:
Sounds good. I decided to remove the `to-be-deprovisioned`. LMK what you think.